api/vulnerabilities_host.go

```diff
@@ -195,6 +195,40 @@ func (assessment *HostVulnHostAssessment) VulnerabilityCounts() HostVulnCounts {
 	return hostCounts
 }
 
+// HighestSeverity returns the highest severity level vulnerability
+func HighestSeverity(vulnCounts HostVulnCounts) string {
+	if vulnCounts.Critical != 0 {
+		return "critical"
+	}
+	if vulnCounts.High != 0 {
+		return "high"
+	}
+	if vulnCounts.Medium != 0 {
+		return "medium"
+	}
+	if vulnCounts.Low != 0 {
+		return "low"
+	}
+	return "unknown"
+}
+
+// HighestFixableSeverity returns the highest fixable severity level vulnerability
+func HighestFixableSeverity(vulnCounts HostVulnCounts) string {
+	if vulnCounts.Critical != 0 {
```

Comment:
All of these are wrong.
```suggestion
	if vulnCounts.CritFixable != 0 {
```